lib/rubocop/cop/lint/duplicated_key.rb

```diff
@@ -0,0 +1,37 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for duplicated keys in hash literals.
+      #
+      # This cop mirrors a warning in Ruby 2.2.
+      #
+      # @example
+      #   hash = { food: 'apple', food: 'orange' }
+      class DuplicatedKey < Cop
+        MSG = 'Duplicated key in hash literal'
```

Comment:
Apart from a missing `.` everything looks good. Btw, you can mention the duplicated key in the message I guess.